projects/fishtornado_xr/FishTornado.h

```diff
@@ -127,8 +127,9 @@ class FishTornadoApp
     Ocean                        mOcean;
     Shark                        mShark;
     bool                         mUsePCF                   = true;
-    uint64_t                     mTotalGpuFrameTime        = 0;
-    grfx::PipelineStatistics     mPipelineStatistics       = {};
+    int                                   mViewCount                = 1;
```

Comment:
Odd, the format checker didn't catch the aligned assignment with the lines above and below this section.

Reply:
Fixed this, I think...